lessons/02-html-blocks/index.html

```diff
@@ -0,0 +1,15 @@
+<html>
+    <head>
+<title><h1>My Favorite Cats</h1></title>
```

Comment:
We can't put an `<h1>` here because `<h1>`'s are only allowed inside the `<body>` tag. This `<title>` tag renders to the browser tab so it will be formatted on it's own

Reply:
> We can't put an `<h1>` here because `<h1>`'s are only allowed inside the `<body>` tag. This `<title>` tag renders to the browser tab so it will be formatted on it's own
Ok , i will not use it with <title> tag. Thanks